Sentry/src/main/kotlin/com/infomaniak/core/sentry/SentryConfig.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Infomaniak Core - Android
+ * Copyright (C) 2025 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.core.sentry
+
+import android.app.Application
+import com.infomaniak.core.network.models.exceptions.NetworkException
+import io.sentry.SentryEvent
+import io.sentry.SentryOptions
+import io.sentry.android.core.SentryAndroid
+import io.sentry.android.core.SentryAndroidOptions
+import io.sentry.android.fragment.FragmentLifecycleIntegration
+import io.sentry.android.fragment.FragmentLifecycleState
+import kotlin.coroutines.cancellation.CancellationException
+
+object SentryConfig {
+    fun Application.configureSentry(
+        isDebug: Boolean,
+        isSentryTrackingEnabled: Boolean,
+        isErrorException: (Throwable?) -> Boolean
```

Comment:
```suggestion
        isErrorException: (Throwable?) -> Boolean = { false },
```

You should set a default value here so you don’t have to redefine it in all projects that don’t have any special exceptions.